pkg/authorization/registry/selfsubjectrulesreview/storage.go

```diff
@@ -79,19 +77,3 @@ func (r *REST) Create(ctx apirequest.Context, obj runtime.Object, _ bool) (runti
 
 	return ret, nil
 }
-
-func (r *REST) filterRulesByScopes(rules []rbac.PolicyRule, scopes []string, namespace string) ([]rbac.PolicyRule, error) {
```

Comment:
huh, I wonder how it's working.

Reply:
@deads2k kind find any reference of where this func is used...